prover/src/constraints/evaluator/lagrange.rs

```diff
@@ -49,6 +50,7 @@ impl<E: FieldElement> LagrangeKernelConstraintsBatchEvaluator<E> {
     ///
     /// Writes the evaluations in `combined_evaluations_acc` at the corresponding (constraint
     /// evaluation) domain index.
+    #[instrument(skip_all)]
     pub fn evaluate_constraints<T>(
```

Comment:
Is there a way to customize how this function appears in the logs? For example, right now it appears in the logs as `evaluate_constraints` but would be great if it appeared as `evaluate_lagrange_kernel_constraints` or something similar.